Website About/OLD/index.html

```
@@ -1,320 +1,320 @@
+    delay="2000"
+    colors="primary:#2d87ff"
+    style="width:24px;height:24px">
+  </lord-icon>
+  Acessar Site
+</button>
+                </div>
+
+        <div class="footer">
+            © 2025 ErikrafT Drop • Todos os direitos reservados
+        </div>
+    </div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)